src/Adapter/MSTestAdapter.PlatformServices/AssemblyResolver.cs

```diff
@@ -358,8 +358,6 @@ protected virtual
             return null;
         }
 
-        DebugEx.Assert(requestedName != null && !StringEx.IsNullOrEmpty(requestedName.Name), "MSTest.AssemblyResolver.OnResolve: requested is null or name is empty!");
```

Comment:
This line was asserting two two conditions. Did you intend to only remove the `requestedName != null`?

Reply:
shit. yes, will ifx